schedule-works-be/dgraph.py

```diff
@@ -1,67 +1,32 @@
-"""Functions for interacting with directed graph structure."""
```

Comment:
We should keep all of this stuff and just implement the new method

Reply:
The topological_sort method I added?